Cargo.lock

```diff
@@ -4837,6 +4825,25 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "sqlparser"
+version = "0.36.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "2eaa1e88e78d2c2460d78b7dc3f0c08dbb606ab4222f9aff36f420d36e307d87"
+dependencies = [
+ "log",
+]
+
+[[package]]
+name = "sqlparser"
+version = "0.37.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "37ae05a8250b968a3f7db93155a84d68b2e6cea1583949af5ca5b5170c76c075"
+dependencies = [
+ "log",
+ "serde",
+]
```

Comment:
As apparently `polars 0.32.1` is still using `sqlparser 0.36.1`, could you use `0.36.1` as well to not duplicate the dependency? In the same vein `nu-command` still has a dependency on `sqlparser 0.34`.

https://github.com/nushell/nushell/blob/1f06f8405c8e2553574a8327944894722e147b9d/crates/nu-command/Cargo.toml#L83

Reply:
Will do